kvdb-rocksdb/src/lib.rs

```diff
@@ -597,6 +597,22 @@ impl Database {
 			.unwrap_or(0)
 	}
 
+	/// The number of keys in a column (estimated).
+	/// Does not take into account the unflushed data.
+	pub fn num_keys(&self, col: u32) -> io::Result<u64> {
+		const ESTIMATE_NUM_KEYS: &str = "rocksdb.estimate-num-keys";
+		match *self.db.read() {
+			Some(ref cfs) => {
+				let cf = cfs.cf(col as usize);
+				match cfs.db.property_int_value_cf(cf, ESTIMATE_NUM_KEYS) {
+					Ok(estimate) => Ok(estimate.unwrap_or_default()),
```

Comment:
hmm, I'm not sure that I follow the propagation of errors here.

If no db exist => Ok(0)
Ok(None) => only happens on bad &str?! 
Err(e) => parsing of number failed or something else? 

according to https://github.com/facebook/rocksdb/blob/08809f5e6cd9cc4bc3958dd4d59457ae78c76660/include/rocksdb/db.h#L654-L689 "rocksdb.estimate-num-keys" will succeed so I see no problem with but some errors might get `swallowed`